support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/serializer/TicketSerializer.java

```diff
@@ -0,0 +1,37 @@
+package org.apereo.cas.serializer;
+
+import org.apereo.cas.ticket.Ticket;
+
+/**
+ * This is {@link TicketSerializer}.
+ *
+ * Abstraction layer for ticket serialization methods
+ *
+ * @author David Rodriguez
+ *
+ * @param <T> should be something that can be written
+ * to Cassandra like {@link String}
+ *
+ * @since 5.2.0
+ */
+public interface TicketSerializer<T> {
```

Comment:
I recommend you remove this interface and take a look at existing serializers that deal with tickets. An example of this would be the mongodb ticket registry for instance.